examples/models/llama2/CMakeLists.txt

```diff
@@ -54,21 +54,15 @@ find_package(executorch CONFIG REQUIRED)
 # llama_runner library
 add_subdirectory(runner)
 
-set(link_options)
 set(link_libraries)
 
 if(EXECUTORCH_BUILD_OPTIMIZED)
-  list(APPEND link_libraries optimized_native_cpu_ops_lib optimized_kernels portable_kernels)
-  list(APPEND link_options
-                      "SHELL:LINKER:--whole-archive \
-                      $<TARGET_FILE:optimized_native_cpu_ops_lib> \
-                      LINKER:--no-whole-archive")
+  list(APPEND link_libraries optimized_native_cpu_ops_lib optimized_kernels
+    portable_kernels)
```

Comment:
portable kernels is linked regardless of optimized? Can we move this outside of the if?

Reply:
Hmn, `optimized_native_cpu_ops_lib` contains portable and optimized kernel registrations. If we don't have an optimized version of an op that's used by the model, we take the portable one.